### ISSUE TITLE
Symfony.Formatting.BlankLineBeforeReturn too strict

### ISSUE THREAD

**Issue by rarila** [djoos/Symfony-coding-standard#203]
Symfony.Formatting.BlankLineBeforeReturn is too strict, as it does allow simple comments, but no analyser instructions.

Here some examples with resulting tokens:

```php
/**
 * Missing blank line before return statement
 * Token: [0 => 'T_PHPCS_IGNORE']
 */
function fails1(): string
{
    // phpcs:ignore Foobar.Whatever.Rule
    return foobar(1);
}
```

```php
/**
 * Missing blank line before return statement
 * Token: [0 => 'T_PHPCS_IGNORE']
 */
function fails2(): string
{
    /* phpcs:ignore Foobar.Whatever.Rule */
    return foobar(1);
}
```

```php
/**
 * Missing blank line before return statement
 * Token: [0 => 'T_PHPCS_IGNORE', 1 => 'T_DOC_COMMENT_OPEN_TAG']
 */
function fails3(): string
{
    /** phpcs:ignore Foobar.Whatever.Rule */
    return foobar(1);
}
```

```php
/**
 * Missing blank line before return statement
 * Token: [0 => 'T_DOC_COMMENT_STRING', 1 => 'T_DOC_COMMENT_TAG', 2 => 'T_DOC_COMMENT_OPEN_TAG'']
 */
function fails4(): string
{
    /** @psalm-var ClassName $variable */
    return foobar(1);
}
```

I guess, given tokens should be added there:

https://github.com/djoos/Symfony-coding-standard/blob/6ab870a3aa5b93dd2201b82f058be4bd54d9b2da/Symfony/Sniffs/Formatting/BlankLineBeforeReturnSniff.php#L79-L84
